sdk/core/core/inc/az_str.h

```diff
@@ -27,10 +28,21 @@
 //
 // where
 //
-// `void some_function(az_const_span const span);`
-#define AZ_STR(STRING_LITERAL) (az_const_span) AZ_CONST_STR(STRING_LITERAL)
+// `void some_function(az_span const span);`
+#define AZ_STR(STRING_LITERAL) (az_span) AZ_CONST_STR(STRING_LITERAL)
 
-extern az_const_span const AZ_STR_ZERO;
+/**
+ * A callback that accepts zero-terminated string `char const *`.
+ */
+AZ_ACTION_TYPE(az_str_action, char const *)
+
+#define AZ_CR '\r'
+
+#define AZ_LF '\n'
+
+#define AZ_CRLF "\r\n"
```

Comment:
Can you see if you can define and use AZ_STR_CRLF the same way we define AZ_STR_ZERO?

Reply:
I need this as a string as well so I can concatenate strings at compile time.